### ISSUE TITLE
Bug: Cannot install on Ubuntu 24.04 

### ISSUE THREAD

**Issue by willy553** [abraunegg/onedrive#2726]
### Describe the bug

I cannot install on ubuntu 24.04. When trying to run `sudo apt install --no-install-recommends --no-install-suggests onedrive`. The following packages have unmet dependencies: `onedrive : Depends: libphobos2-ldc-shared100 (>= 1:1.30.0) but it is not installable`. 

### Operating System Details

```shell
Linux williams-laptop 6.8.0-31-generic #31-Ubuntu SMP PREEMPT_DYNAMIC Sat Apr 20 00:40:06 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

Distributor ID:	Ubuntu
Description:	Ubuntu 24.04 LTS
Release:	24.04
Codename:	noble
```

### Client Installation Method

From 3rd Party Source (PPA, OpenSuSE Build Service etc)

### OneDrive Account Type

Business | Office365

### What is your OneDrive Application Version

N/A

### What is your OneDrive Application Configuration

```shell
N/A
```

### What is your 'curl' version

```shell
N/A
```

### Where is your 'sync_dir' located

Local

### What are all your system 'mount points'

```shell
N/A
```

### What are all your local file system partition types

```shell
N/A
```

### How do you use 'onedrive'

N/A

### Steps to reproduce the behaviour

Follow instructions to install for Ubuntu 23.10 (Which is the closes available version to 24.04)

### Complete Verbose Log Output

```shell
N/A
```

**Comment by abraunegg** [comment]
@willy553 
> I cannot install on ubuntu 24.04.

A simple search of [Discussions](https://github.com/abraunegg/onedrive/discussions) would have yielded you the information you are after: https://github.com/abraunegg/onedrive/discussions/2706

> Follow instructions to install for Ubuntu 23.10 (Which is the closes available version to 24.04)

I am curious why you thought this would even work. They are totally different releases.

Please do a little research next time before opening a bug report (which is not even a bug) .....

**Comment by abraunegg** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
